### PR TITLE
Tidy statements

### PR DESCRIPTION
**Pull request by baurine**
Done #436 

What did:

- Remove un-necessary props
- Remove everything about instance
- Use session storage to replace context
- Save desc and order options as well likes SlowQuery
- Extract useStatement()
- Refine TimeRangeSelector
- Adjust files structure
- Refine backend, use `ShouldBindQuery()`
- Adjust slow query files structure

### CODE REVIEW

File: ui/lib/apps/SlowQuery/utils/useSlowQuery.ts [hunk 1]
```diff
@@ -17,34 +17,38 @@ export interface IQueryOptions {
...
 
 export default function useSlowQuery(
-  options?: IQueryOptions,
+  options?: ISlowQueryOptions,
   needSave: boolean = true
 ) {
-  const [queryOptions, setQueryOptions] = useState(() =>
-    options ? options : getDefQueryOptions()
+  const [queryOptions, setQueryOptions] = useState(
+    () => options || DEF_SLOW_QUERY_OPTIONS
```
Comment (baurine) [inline ui/lib/apps/SlowQuery/utils/useSlowQuery.ts:36]: no need method, `options || DEF_SLOW_QUERY_OPTIONS` is fine.

**Comment by baurine** [comment]
@breeswish @Deardrops PTAL.

File: ui/lib/apps/Statement/components/StatementsTable.tsx [hunk 2]
```diff
@@ -80,32 +69,58 @@ interface Props extends Partial<ICardTableV2Props> {
...
 }: Props) {
   const navigate = useNavigate()
-  const [items, setItems] = useState(statements)
-  const [columns, setColumns] = useState(
-    tableColumns(statements, onColumnClick, showFullSQL)
-  )
+
+  // const [columns, setColumns] = useState(
+  //   tableColumns(statements, onColumnClick, orderBy, desc, showFullSQL)
+  // )
```
Comment (Deardrops) [inline ui/lib/apps/Statement/components/StatementsTable.tsx:96]: Seems we do not need these code any more.

**Comment by breeswish** [comment]
Cool! You can rebase this to master

**Comment by baurine** [comment]
> Cool! You can rebase this to master

Done! please take a look again, thank you! @breeswish

**Pull request merged**